### PR TITLE
Switch to pretty_assertions

### PR DESCRIPTION
**Pull request by djmitche**
This is a big diff but it's very formulaic.  I basically added pretty_assertions to every file with `mod test {` in it, and then ran `cargo fix` to remove the unnecessary imports.

### CODE REVIEW

File: taskchampion/src/workingset.rs [hunk 1]
```diff
@@ -69,6 +69,7 @@ impl WorkingSet {
 #[cfg(test)]
 mod test {
     use super::*;
+    use pretty_assertions::{assert_eq};
```
Comment (dbr) [inline taskchampion/src/workingset.rs:72]: I'm surprised `cargo fmt` didn't remove the redundant `{ ... }` brackets?

**Review by dbr — APPROVED** [review]
LGTM repeated 47 times 😁

File: taskchampion/src/workingset.rs (hunk 1 above)
Comment (djmitche) [reply taskchampion/src/workingset.rs:72]: Good point. I'm also surprised that didn't cause a CI failure.  I'll sed that out and land.

**Comment by dbr** [comment]
I wonder if the formatting checks are not running on the `#[test]` targets?

**Comment by djmitche** [comment]
It appears we're not actually running `cargo fmt` in CI.  I'll make another PR :)

**Pull request merged**